src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs

```diff
@@ -4,16 +4,23 @@
     Name = "Localization",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
-    Version = ManifestConstants.OrchardCoreVersion,
+    Version = ManifestConstants.OrchardCoreVersion
+)]
+
+[assembly: Feature(
+    Id = "OrchardCore.Localization",
+    Name = "Localization",
     Description = "Provides support for UI localization.",
-    Category = "Internationalization",
-    Dependencies = new[] { "OrchardCore.Settings" }
+    Dependencies = new[] { "OrchardCore.Settings" },
+    Category = "Internationalization"
 )]
-#if (NET5_0 || NET5_0_OR_GREATER)
+
+#if NET5_0_OR_GREATER
```

Comment:
@jtkech are you sure this work at your side, I presume the `NET5_0_OR_GREATER` is not supported yet

Reply:
@hishamco Yes it is, just tried through the donet cli

It compiles well under 3.1 but only if we use the directive, so the directive works.

Reply:
I will try it from VS 2019, coz the code is grayed out at the time of pushing the PR